### DISCUSSION TITLE
How to join the slack group?

### DISCUSSION THREAD

**Discussion by KingsonKai** [chaos-mesh/chaos-mesh#3951] [Q&A]
https://communityinviter.com/apps/cloud-native/cncf
After entering this link, after entering my email, I was prompted with this:

![image](https://user-images.githubusercontent.com/27922190/222319441-ff905338-b87e-4a59-be4a-81eb92818911.png)

**Comment by fewdan** [comment]
I tried it, maybe it has something to do with your email address.
If I use QQ email, "bad_invite_domain" will pop up, but if I use some other emails (such as outlook/gmail), it can be used normally.

**Reply by fewdan** [reply]
@KingsonKai So, maybe you can try to get into the slack group with another email address.

**Reply by KingsonKai** [reply]
Thanks, I'll try it
